src/Microsoft.DocAsCode.Dfm/MarkdownValidators/MarkdownValidatorBuilder.cs

```diff
@@ -82,6 +125,30 @@ public IMarkdownTokenRewriter Create()
                 });
         }
 
+        private IEnumerable<MarkdownTagValidationRule> GetEnabledTagRules()
+        {
+            foreach (var item in _tagValidators)
+            {
+                if (item.FullId != null)
+                {
+                    MarkdownValidationRule rule;
+                    if (_validators.TryGetValue(item.FullId, out rule) ||
+                        _validators.TryGetValue(item.Category, out rule))
```

Comment:
rule name can be category name or fullid

---
In reply to: [72756801](https://github.com/dotnet/docfx/pull/545#discussion_r72756801) [](ancestors = 72756801)